.github/tools/run_black.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# Exit on error, print commands
+set -ex
```

Comment:
Only suggestion is  that 

```
set -o xtrace
set -o errexit
```

is clearer.

Reply:
Usually see the short version in system scripts, so I didn't even think of the possibility